### PR TITLE
Hardware overview

### PR DESCRIPTION
**Pull request by Emethium**
Hardware overview section for the docs

### CODE REVIEW

File: sections/overview/graphics.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## The Graphics System
+
+- The GBA refresh rate equates do 280,896 clock cycles per frame (the time it takes to display the entire video buffer) which equals to **59.73 Hz**, our maximum frame rate.
```
Comment (alvadorn) [inline sections/overview/graphics.md]: `equates to` ?

**Review by alvadorn — CHANGES_REQUESTED** [review]

File: sections/overview/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- **Sound:** 4 sounds (corresponding to GBC sounds) + 2 CPU direct sounds (PCM format)
+
+### Retrocompatibility
+
+- The Game Boy Advance incorporated the original Z80 CPU from the old Game Boy Color, providing complete backward compatibility with all previous gameboy games.
+- The GBA auto-detects the type of cartridge that has been inserted and either boots up the ARM7 or the Z80 CPU
+
+### Hardware Access
+
+- The GBA architecture is similar to a PC in many ways. Both have one or more processors, RAM, a digital signal processor (DSP), a motherboar and a BIOS (Basic Input/Output System) that causes the hardware to boot up.
```
Comment (alvadorn) [inline sections/overview/index.md]: `a motherboard`

**Review by alvadorn — CHANGES_REQUESTED** [review]

File: sections/overview/index.md (hunk 2 above)
Comment (Emethium) [reply sections/overview/index.md]: Typo fixed.

File: sections/overview/graphics.md (hunk 1 above)
Comment (Emethium) [reply sections/overview/graphics.md]: Yeah, another typo :sweat: Fixed!

**Review by alvadorn — APPROVED** [review]

**Pull request merged**